client/packages/common/src/types/schema.ts

```diff
@@ -3160,6 +3160,7 @@ export type NameNode = {
   comment?: Maybe<Scalars['String']['output']>;
   country?: Maybe<Scalars['String']['output']>;
   createdDatetime?: Maybe<Scalars['DateTime']['output']>;
+  customData?: Maybe<Scalars['JSON']['output']>;
```

Comment:
offroad: I've re-generated, and the generate wasn't called after the custom data was added to names